### PR TITLE
PIMS-973 add paginated access request list.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: backend/entities/AccessRequest.cs [hunk 1]
```diff
@@ -44,14 +44,14 @@ public class AccessRequest : BaseEntity
         /// get - the list of agencies that the user is requesting to be added to.
         /// </summary>
         /// <returns></returns>
-        public ICollection<AccessRequestAgency> Agencies { get; } = new List<AccessRequestAgency>();
+        public ICollection<AccessRequestAgency> Agencies { get; private set; } = new List<AccessRequestAgency>();
```
Comment (Fosol) [inline backend/entities/AccessRequest.cs]: Not sure the value of a `private set` for EF entities.

File: backend/test/Controllers/Admin/UserControllerTest.cs [hunk 2]
```diff
@@ -71,6 +55,25 @@ public UserControllerTest()
             _userController = _helper.CreateAdminUserController(user);
             _mapper = _helper.GetService<IMapper>();
             _pimsService = _helper.GetService<Mock<IPimsAdminService>>();
+
+            _expectedAccessRequest.Agencies.Add(new AccessRequestAgency()
```
Comment (Fosol) [inline backend/test/Controllers/Admin/UserControllerTest.cs]: I did the same thing in my PR :P

File: frontend/src/constants/reducerTypes.tsx [hunk 3]
```diff
@@ -10,5 +10,7 @@ export const LOOKUP_CODE = 'lookupCode';
 export const GET_LOOKUP_CODES = 'GET_LOOKUP_CODES';
 
 // Auth
+export const ACCESS_REQUEST = 'accessRequest';
```
Comment (Fosol) [inline frontend/src/constants/reducerTypes.tsx]: What are these consts for?

File: backend/entities/AccessRequest.cs (hunk 1 above)
Comment (devinleighsmith) [reply backend/entities/AccessRequest.cs]: Profile wasn't working without this.

File: frontend/src/constants/reducerTypes.tsx (hunk 3 above)
Comment (devinleighsmith) [reply frontend/src/constants/reducerTypes.tsx]: consistent use throughout the application. This value is needed by the reducer and also by the selector, and action. Using the constants saves me from having to remember the keys these values are saved/retrieved by.

File: frontend/package.json [hunk 4]
```diff
@@ -6,6 +6,7 @@
     "@bcgov/bc-sans": "1.0.1",
     "@react-keycloak/web": "1.0.0-alpha.6",
     "@reduxjs/toolkit": "1.2.3",
+    "@types/react-paginate": "6.2.1",
```
Comment (asanchezr) [inline frontend/package.json]: ^ The type reference should be a **devDependency** as it only provides intellisense and type support for TypeScript. Just a minor tweak that can be dealt as tech-debt later

File: frontend/src/reducers/accessRequestReducer.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+const initialState: IAccessRequestState = {
+  pagedAccessRequests: { page: 0, total: 0, quantity: 0, items: [] },
+};
+
+const accessRequestReducer = (state = initialState, action: IStoreAccessRequestsAction) => {
+  switch (action.type) {
+    case actionTypes.STORE_ACCESS_REQUESTS:
+      console.log('storing access request');
```
Comment (asanchezr) [inline frontend/src/reducers/accessRequestReducer.tsx]: /nit console.logs are fine for debugging but we should try to take them out out public PRs. We might need to automate this as myself keep forgetting to remove them too!

**Pull request merged**